### DISCUSSION TITLE
Some new feature suggestions on iPadOS

### DISCUSSION THREAD

**Discussion by SuperKenVery** [saber-notes/saber#692] [Ideas]
Some of my thoughts to add to the roadmap:
- Bezier curve, to make the line look more natural
- Apple Pencil has a higher refresh rate than touch screen, and we should make use of that

These are very common things to have for a note app on iOS. 

And some longer-term targets I would propose:
- Infinite large paper, both horizontal and vertical, just like OneNote. 

I really appreciate your work! I've been wanting an open source note app on iPad and finally here's one!

**Comment by neakmenter** [comment]
Hi, I hope no-one minds if I add my iOS related thoughts on this great app here?

- Could we have Apple Pencil eraser select (on double tap on the pencil body)?
- Also, perhaps an onscreen ruler/protractor may be useful sometimes?

Thanks for being pretty much the only open source cross platform handwriting notes app that includes an iOS version!

**Reply by DocFreeman** [reply]
Agreed—not being able to use the squeeze and double tap functionality in app is a bit of a drag but again, there is no other open source app that is this robust. Thanks to all contributors.
